tests/component/test_task_read_waveform_di.py

```diff
@@ -0,0 +1,176 @@
+from __future__ import annotations
+
+import numpy
+import pytest
+from nitypes.waveform import DigitalWaveform
+
+import nidaqmx
+import nidaqmx.system
+from nidaqmx.constants import AcquisitionType, LineGrouping
+from nidaqmx.utils import flatten_channel_string
+
+
+@pytest.fixture
+def di_single_channel_timing_task(
+    task: nidaqmx.Task, sim_6363_device: nidaqmx.system.Device
+) -> nidaqmx.Task:
+    task.di_channels.add_di_chan(
+        sim_6363_device.di_lines[0].name, line_grouping=LineGrouping.CHAN_FOR_ALL_LINES
+    )
+    task.timing.cfg_samp_clk_timing(1000.0, sample_mode=AcquisitionType.FINITE, samps_per_chan=50)
+    return task
+
+
+@pytest.fixture
+def di_multi_channel_timing_task(
+    task: nidaqmx.Task, sim_6363_device: nidaqmx.system.Device
+) -> nidaqmx.Task:
+    task.di_channels.add_di_chan(
+        flatten_channel_string(sim_6363_device.di_lines.channel_names[:8]),
```

Comment:
@mikeprosserni @jfriedri-ni Please test a port channel as well. I think the line order is reversed.

This is related to [AB#3178052](https://dev.azure.com/ni/DevCentral/_workitems/edit/3178052).

Reply:
Thanks, I overlooked this because "Large diffs are not rendered by default."